kubemarine/core/errors.py

```diff
@@ -89,6 +89,10 @@ def get_kme_dictionary() -> dict:
                     "in cluster.yaml{previous_version_spec}, "
                     "but not present in procedure inventory{next_version_spec}. "
                     "Please, specify required 'sandbox_image' explicitly in procedure inventory."
+        },
+        'KME0014': {
+            "name": "Provided helm chart url {url} returns not a {type} content. Please check, that is returned in "
```

Comment:
@disa1217  pls update

Reply:
@koryaga , reviewed.

Reply:
Pls also check that after your changes CI is completed successfully